app/presenters/concerns/hyrax/iiif_av/displays_content.rb

```diff
@@ -78,12 +78,14 @@ def video_content
           # @see https://github.com/samvera-labs/iiif_manifest
           if solr_document['derivatives_metadata_ssi'].present?
             files_metadata = JSON.parse(solr_document['derivatives_metadata_ssi'])
-            external_files = files_metadata.select { |f| f['external_file_uri'].present? }
+            external_files = files_metadata.select { |f| f['file_location_uri'].present? }
```

Comment:
Since it's no longer `external_files`, should we name it something like `file_locations`?

Reply:
Nice catch.